draft-ietf-httpbis-priority.md

```diff
@@ -108,35 +110,34 @@ The term control stream is used to describe the HTTP/2 stream with identifier
 0x0, and HTTP/3 control stream; see {{Section 6.2.1 of !HTTP3=I-D.ietf-quic-http}}.
 
 
-# Motivation for Replacing HTTP/2 Priorities {#motivation}
+# Motivation for Replacing RFC 7540 Priorities {#motivation}
 
 An important feature of any implementation of a protocol that provides
 multiplexing is the ability to prioritize the sending of information. This was
-an important realization in the design of HTTP/2. Prioritization is a
-difficult problem, so it will always be suboptimal, particularly if one endpoint
-operates in ignorance of the needs of its peer.
-
-HTTP/2 introduced a complex prioritization scheme that uses a combination of
-stream dependencies and weights to describe an unbalanced tree. This scheme has
-suffered from poor deployment and interoperability.
-
-Clients build an HTTP/2 prioritization tree through a series of individual
-stream relationships, which are transferred to the server using HTTP/2 priority
-signals in either of three forms. First, a HEADERS frame with the PRIORITY flag
-set is an explicit signal that includes an Exclusive flag, Stream Dependency
-field, and Weight field. Second, a HEADERS frame with no PRIORITY flag is an
-implicit signal to use the default priority. Third, the PRIORITY frame, which is
-always explicit since it always contains an Exclusive flag, Stream Dependency
-field, and Weight field.
-
-The rich flexibility of tree building is rarely exercised. Experience has shown
-that clients tend to choose a single model optimized for a web use case and
-experiment within the model constraints, or do nothing at all. Furthermore, many
-clients build their prioritization tree in a unique way, which makes it
-difficult for servers to understand their intent and act or intervene
-accordingly.
-
-Many HTTP/2 server implementations do not include support for the priority
+an important realization in the design of HTTP/2 leading to RFC 7540.
+Prioritization is a difficult problem, so it will always be suboptimal,
+particularly if one endpoint operates in ignorance of the needs of its peer.
+
+RFC 7540 {{?RFC7540}} introduced a complex prioritization scheme that uses a
+combination of stream dependencies and weights to describe an unbalanced tree.
+This scheme suffered from poor deployment and interoperability, it was
+deprecated in HTTP/2 ({{HTTP2}}). However, in order to maintain wire
+compatibility, HTTP/2 priority signals are still mandatory to handle. These come
+in three forms. First, a HEADERS frame {{Section 6.2 of HTTP2}} with the
+PRIORITY flag set is an explicit signal that includes an Exclusive flag, Stream
+Dependency field, and Weight field. Second, a HEADERS frame with no PRIORITY
```

Comment:
You can probably skip the detailed description of fields and frames.

Reply:
@mnot asked for an explicit definition of what we meant by HTTP/2 signals  a little while ago. I think it adds value.

Reply:
I've actually done this, preferring to delegate to https://www.ietf.org/archive/id/draft-ietf-httpbis-http2bis-03.html#section-5.3.2